pallets/simple-nft/src/weights.rs

```diff
@@ -27,18 +27,16 @@ pub trait WeightInfo {
 /// Weight functions for pallet_simple_nft.
 pub struct SubstrateWeight<T>(PhantomData<T>);
 impl<T: frame_system::Config> WeightInfo for SubstrateWeight<T> {
-    fn run_process(i: usize, o: usize) -> Weight {
-        (19_061_000 as Weight)
-            // Standard Error: 1_472_000
-            .saturating_add((17_847_000 as Weight).saturating_mul(i as Weight))
-            // Standard Error: 1_472_000
-            .saturating_add((8_875_000 as Weight).saturating_mul(o as Weight))
-            .saturating_add(T::DbWeight::get().reads(1 as Weight))
-            .saturating_add(T::DbWeight::get().reads((1 as Weight).saturating_mul(i as Weight)))
-            .saturating_add(T::DbWeight::get().writes(1 as Weight))
-            .saturating_add(T::DbWeight::get().writes((1 as Weight).saturating_mul(i as Weight)))
-            .saturating_add(T::DbWeight::get().writes((1 as Weight).saturating_mul(o as Weight)))
-    }
+	fn run_process(i: usize, o: usize) -> Weight {
+		(104_267_000 as Weight)
+			// Standard Error: 6_647_000
+			.saturating_add((13_768_000 as Weight).saturating_mul(i as Weight))
+			.saturating_add(T::DbWeight::get().reads(1 as Weight))
+			.saturating_add(T::DbWeight::get().reads((1 as Weight).saturating_mul(i as Weight)))
+			.saturating_add(T::DbWeight::get().writes(1 as Weight))
+			.saturating_add(T::DbWeight::get().writes((1 as Weight).saturating_mul(i as Weight)))
+			.saturating_add(T::DbWeight::get().writes((1 as Weight).saturating_mul(o as Weight)))
+	}
```

Comment:
This is suspicious. Specifically that the base weight has gone up by so much and there is now no element based on processing the each output like ` .saturating_add((8_875_000 as Weight).saturating_mul(o as Weight))`. I think we can let this pass, but we should look into improving these as time goes on